pootle/static/js/editor/components/SuggestionFeedbackForm.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+
+import FormElement from 'components/FormElement';
+import FormMixin from 'mixins/FormMixin';
+
+import { decodeEntities } from '../utils';
+
+export const SuggestionFeedBackForm = React.createClass({
+
+  propTypes: {
+    suggId: React.PropTypes.number.isRequired,
+    suggText: React.PropTypes.string.isRequired,
+    editor: React.PropTypes.object.isRequired,
```

Comment:
Yes, passing callbacks as props is the recommended approach for parent-child communication. Please also note to [name methods passed via props properly](http://docs.translatehouse.org/projects/pootle/en/latest/developers/styleguide.html#react-jsx): `handle*()` for methods, `on*()` for props.